src/events.test.js

```diff
@@ -107,5 +109,17 @@ describe('events', () => {
       arn: 'arn:aws:s3:::mybucket',
       triggeredBy: 's3',
     });
+
+    jest.spyOn(utils, 'isStepFunction');
```

Comment:
Don't mock this function, mock low-level as you can, override ENV var or something